source/IOSResolver/src/IOSResolver.cs

```diff
@@ -149,7 +153,23 @@ private class Pod {
                 var outputPropertiesByName = new Dictionary<string, string>(propertiesByName);
                 var path = LocalPath;
                 if (!String.IsNullOrEmpty(path)) {
-                    outputPropertiesByName["path"] = String.Format("'{0}'", Path.GetFullPath(path));
+                    if(outputPropertiesByName.ContainsKey("podspecPath")) {
+                        // Check if it is a local folder or http://
+                        // If it is local folder add full path into OutputProperties
+                        // If it is URI, kept as-is
+                        if(Uri.IsWellFormedUriString(path, UriKind.Absolute)) {
+                            outputPropertiesByName.Add("podspec", String.Format("'{0}'", path));
+                        }
+                        else {
+                            outputPropertiesByName.Add("podspec", String.Format("'{0}'", Path.GetFullPath(path)));
+                        }                      
+                        // Remove "path" parameter and temporary "podspecPath" parameter
+                        outputPropertiesByName.Remove("path");  
+                        outputPropertiesByName.Remove("podspecPath");  
+                    }
+                    else {
+                        outputPropertiesByName["path"] = String.Format("'{0}'", Path.GetFullPath(path));
+                    }                    
```

Comment:
Well, it is really nice to check redundancy here but I will leave that responsibility to the person who author `Dependencies.xml`, unless you are adding additional warning here when both properties exist.

I think it is much cleaner for you and your case now to just treat them as separate properties, like this
```
                var podSpecPath = PodSpecPath;
                if (!String.IsNullOrEmpty(podSpecPath)) {
                    if(Uri.IsWellFormedUriString(podSpecPath, UriKind.Absolute)) {
                        outputPropertiesByName.Add("podspec", String.Format("'{0}'", podSpecPath));
                    }
                    else {
                        outputPropertiesByName.Add("podspec", String.Format("'{0}'", Path.GetFullPath(podSpecPath)));
                    }
                }

                var localPath = LocalPath;
                if (!String.IsNullOrEmpty(localPath)) {
                    outputPropertiesByName["path"] = String.Format("'{0}'", Path.GetFullPath(localPath));
                }

```